bundles/org.openhab.binding.samsungtv/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -34,7 +34,7 @@
 				<option value="WebSocket">Websocket (2016 and later TV's)</option>
 				<option value="SecureWebSocket">Secure websocket (2016 and later TV's)</option>
 			</options>
-			<default>SecureWebSocket</default>
+			<default>None</default>
```

Comment:
What does „None“ mean? Will it work at all?

Reply:
That none of the other protocols for remote is detected/used. This will happen for 2014/2015 models. Volume and mute channels still will work as these use upnp.